config/epimodels.yml

```diff
@@ -0,0 +1,68 @@
+# Prefix for container names; we'll use {container_prefix}-(container_name)
+container_prefix: epimodels
+
+# Docker org for images
+repo: mrcide
+
+network: wodin-nw
+
```

Comment:
for now, no hostname config option, this will be added when I add the SSL cert and key functionaility!